### ISSUE TITLE
Is this maintained?

### ISSUE THREAD

**Issue by zachChilders** [boncheolgu/tflite-rs#31]
Hello - it looks like version 0.9.0 doesn't build correctly because of an issue in build.rs.  I can definitely put in some work to get it running, but wanted to know if this is being maintained long term before I do.  It looks like it's been broken for several months as is.

**Comment by boncheolgu** [comment]
Hi, I have no problem in using v0.9.0. If you found any issue, please let me know. Contributing is always welcome. Thanks for the interest.

**Comment by fariadif** [comment]
I found some issues running the crate as well. The error is the following:
```
error[E0063]: missing field `content_only` in initializer of `fs_extra::dir::CopyOptions`
  --> /X/X/.cargo/registry/src/github.com-1ecc6299db9ec823/tflite-0.9.0/build.rs:26:20
   |
26 |     let copy_dir = fs_extra::dir::CopyOptions {
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `content_only`

error: aborting due to previous error
```
The problem seems to be in the `fs_extra` version, which was updated recently from `1.1.0` to `1.2.0`.
I managed to change back to `1.1.0` and it worked perfectly.

**Comment by zachChilders** [comment]
@fariadif that's the exact error I'm seeing - I'll give that a shot on my end, this would save me a lot of effort :)

**Comment by boncheolgu** [comment]
@zachChilders @fariadif 
The issue was resolved in v0.9.1, I just published a few hours ago.
Thanks for the report.
